sats-dna/src/main/kotlin/com/sats/dna/components/sessiondetails/SessionDetailsInfoLabel.kt

```diff
@@ -36,6 +37,8 @@ fun SessionDetailsInfoSection(
     dateLabel: @Composable () -> Unit,
     locationLabel: @Composable () -> Unit,
     modifier: Modifier = Modifier,
+    workoutTypeLabel: (@Composable () -> Unit)? = null,
```

Comment:
@sindrenm thoughts on what slots should be optional here?

Reply:
I'm kinda thinking none of them needs to be optional. We can explicitly pass in `null` for those cases where we don't have them, IMO. :+1: